mscore/editstyle.ui

```diff
@@ -706,7 +706,7 @@
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <property name="verticalSpacing">
-            <number>-1</number>
+            <number>6</number>
```

Comment:
Please get rid of those -1 to 6 changes, they have their purpose

Reply:
Done.